### DISCUSSION TITLE
[ALL] 1차 기능구현 리스트

### DISCUSSION THREAD

**Discussion by CJC0512** [OmokNoonE/OnionHotSayYo-backend#3] [생각 & 제안]
## BE

- 파이썬 활용 크롤링 및 자바 연동 구현 (https://www.contestkorea.com/)
  - 대회.공모전에 대한 정보 크롤링 (1~2명)
- DB 및 더미 데이터 구현 (1명)
- Spring Data JPA로 CRUD 구현 (2명)

## FE

- 스토리 보드 디자인 수정 (1~2명)
- Vue.js를 활용한 화면 구현 (1명)

**Comment by CJC0512** [comment]
제가 크롤링하겠습니다.

**Comment by jlee38266** [comment]
전 CRUD 구현 부분을 담당할께요

**Comment by EI-FIE** [comment]
더미데이터 마저 작성하고 CRUD 합류하겠습니다

**Comment by ms1011** [comment]
제가 스토리 보드 디자인 수정 후 Vue.js 화면 구현 하겠습니다

**Comment by jihyeongeun** [comment]
Vue.js 화면 구현하도록 하겠습니다!
